package.json

```diff
@@ -1,56 +1,51 @@
 {
   "name": "parrot",
   "version": "3.0.0",
+  "license": "Apache-2.0",
+  "repository": "americanexpress/parrot",
+  "contributors": [
+    "Jack Cross <jack.cross1@aexp.com>",
+    "Nathan Force <nathan.a.force@aexp.com>",
+    "Jason Schapiro"
+  ],
   "scripts": {
     "postinstall": "lerna bootstrap",
     "clean": "git clean -d -X",
     "test": "jest",
     "pretest": "npm run lint",
-    "lint": "eslint --ext js,jsx ./",
-    "precommit": "lint-staged"
-  },
-  "lint-staged": {
-    "*.{js,jsx,json,css,md}": ["prettier --write", "git add"]
+    "lint": "eslint . --ext js,jsx,md"
   },
-  "contributors": [
-    "Jack Cross <jack.cross1@aexp.com>",
-    "Nathan Force <nathan.a.force@aexp.com>",
-    "Jason Schapiro"
-  ],
   "devDependencies": {
-    "amex-jest-preset-react": "^4.0.0",
-    "babel-cli": "^6.7.5",
-    "babel-eslint": "^7.1.1",
-    "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-polyfill": "^6.23.0",
-    "babel-preset-amex": "^1.1.0",
-    "babel-register": "^6.7.2",
-    "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1",
-    "enzyme-to-json": "^3.3.1",
-    "eslint": "^3.0.0",
-    "eslint-config-amex": "^6.0.0",
-    "eslint-config-prettier": "^2.9.0",
-    "husky": "^0.14.3",
+    "@babel/core": "^7.7.7",
+    "@babel/polyfill": "^7.7.0",
+    "@commitlint/cli": "^8.2.0",
+    "@commitlint/config-conventional": "^8.2.0",
+    "@testing-library/react-hooks": "^3.2.1",
+    "amex-jest-preset-react": "^5.0.2",
+    "babel-jest": "^24.9.0",
+    "babel-preset-amex": "^3.3.0",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
+    "enzyme-to-json": "^3.4.3",
+    "eslint": "^6.8.0",
+    "eslint-config-amex": "^11.1.0",
+    "eslint-config-prettier": "^6.7.0",
+    "eslint-plugin-prettier": "^3.1.2",
+    "husky": "^3.1.0",
     "identity-obj-proxy": "^3.0.0",
     "isomorphic-fetch": "^2.2.1",
-    "jest": "^21.0.0",
-    "lerna": "2.1.2",
-    "lint-staged": "^7.0.0",
-    "prettier": "^1.10.2",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0",
-    "rimraf": "^2.2.8",
-    "whatwg-fetch": "^2.0.3"
+    "jest": "^24.9.0",
+    "lerna": "^3.19.0",
```

Comment:
With the update to lerna 3 you can run this https://github.com/lerna/lerna#common-devdependencies `lerna link convert` to consolidate the dev dependencies

Reply:
Good to know! Thank you for sharing